cirq/circuits/qasm_output.py

```diff
@@ -167,6 +167,18 @@ def __init__(
         precision: int = 10,
         version: str = '2.0',
     ) -> None:
+        """Representation of a circuit in QASM format.
+
+        Args:
+            operations: An operation or tree of operations to insert.
```

Comment:
From the type annotation, we should only pass the tree of operations. Please remove `An operation or`

Reply:
Okay, changes are already made.